nwb_conversion_tools/utils/json_schema.py

```diff
@@ -23,41 +23,64 @@ def exist_dict_in_list(d, ls):
     return any([d == i for i in ls])
 
 
-def append_replace_dict_in_list(d, ls, k):
+def append_replace_dict_in_list(d, ls, k, list_dict_deep_update: bool = True):
```

Comment:
@Saksham20 If you add an argument like this, you must use the docstring to explain what you are doing